### PR TITLE
Pr/1

### PR DESCRIPTION
**Pull request by Mohammadreza-Rokni**
please check it

### CODE REVIEW

File: WORDS/main_Services/common_actions_service.py [hunk 1]
```diff
@@ -19,38 +19,58 @@ def get_words_from_usr():
...
-        end = int(pass_length_range[1])
-        if(len(pass_length_range) == 2 and start < end):
+    while True:
+        try:
+            pass_length = input('What is the range length of the password? example 6-12:')
+            start, end = map(int, pass_length.split('-'))
+            if end <= start:
+                raise ValueError
             return start, end
-        print(
```
Comment (BamdadTabari) [inline WORDS/main_Services/common_actions_service.py:30]: why you cleaned this message? always consume the user as a cow. they need instruction

File: WORDS/main_Services/common_actions_service.py [hunk 2]
```diff
@@ -19,38 +19,58 @@ def get_words_from_usr():
...
+                """
+                !error!\n
+                please follow given structure\n
+                type two numbers as range numbers with '-' between them\n
+                Second number should be bigger than first number
+                """
             )
-        sleep(2)
-        get_pass_range_from_user()
-    except:
```
Comment (BamdadTabari) [inline WORDS/main_Services/common_actions_service.py:40]: what the fuck???

File: WORDS/main_Services/common_actions_service.py [hunk 3]
```diff
@@ -19,38 +19,58 @@ def get_words_from_usr():
...
-            please follow given structure\n
-            type two number as range numbers with '-' between them\n
-            second number most be bigger than first number\n
-            dont type anything else
-            """
-            )
-        sleep(2)
-        # recall the function
-        # تابع بازگشتی
-        get_pass_range_from_user()
```
Comment (BamdadTabari) [inline WORDS/main_Services/common_actions_service.py:53]: are you serious?!

**Review by BamdadTabari — COMMENTED** [review]
give me the answer, what are these codes?

File: WORDS/main_Services/common_actions_service.py (hunk 2 above)
Comment (Mohammadreza-Rokni) [reply WORDS/main_Services/common_actions_service.py:40]: i changed the base code and i didn't know that i shouldn't change the functionality of the original code

File: WORDS/main_Services/common_actions_service.py (hunk 1 above)
Comment (mostafakavand) [reply WORDS/main_Services/common_actions_service.py:30]: Well, I have put the same thing again in a different form below

File: WORDS/main_Services/common_actions_service.py (hunk 3 above)
Comment (Mohammadreza-Rokni) [reply WORDS/main_Services/common_actions_service.py:53]: Because there was a problem with my commit inside vs code, I sent PR several times

Comment (BamdadTabari) [reply WORDS/main_Services/common_actions_service.py:53]: this is not a reason. fix your tools, man,

**Review by BamdadTabari — APPROVED** [review]
done

**Pull request merged**